gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Wed May 06 16:47:37 CDT 2015
+#Fri Sep 25 20:01:03 CDT 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-all.zip
```

Comment:
this change was suggested by Android Studio to get better intellisense in gradle build scripts. It just tells gradle wrapper to pull a gradle distribution that includes the gradle sources.